packages/material-ui-codemod/src/v0.15.0/import-path.js

```diff
@@ -1,4 +1,5 @@
 const pathConversion = {
+  alert: 'Alert',
```

Comment:
No needed
```suggestion
```

Reply:
removed in https://github.com/mui-org/material-ui/pull/18702/commits/f217f972843ce74dd02cb12cdac0f286a46b2c68#diff-19bf0e5dcc69095ac3b6192743f39dd0L2

I wasn't sure what this was for - so I added it in an attempt to mitigate things that may go wrong.  What's it for, by the way?

Reply:
This is an old codemod to upgrade from v0.15 to v0.16. I think that the only reason we keep it is that it can help to write future codemod (as source of inspiration).

Reply:
cool.  thanks!